### ISSUE TITLE
witchwood help plz

### ISSUE THREAD

**Issue by Webber88** [Mithion/ArsMagica2#1409]
so i have the 1.4.009 version of am2(the newest version in case i wrote it wrong) and i cant find any witchwood forest biomes...at all. so i turned the rarity up to 100(since it says low numbers means lesser chance of spawning) and still didnt find anything(fyi i looked for an hour) so does anyone recommend a number i could set it to so i could find a witchwood biome?

**Comment by ChilledDayz** [comment]
I have two of them right near spawn on the default  settings. So they are spawning for me.

**Comment by Webber88** [comment]
so... clearly that isnt the case for me... since i have still yet to find any.

**Comment by ChilledDayz** [comment]
Yeah clearly,  are you running any other mods that add biome.

**Comment by bookerthegeek** [comment]
I also thought they were no spawning. Finally found one with just AM2 installed. Could we get a configuration for weight of the biome?

**Comment by ChilledDayz** [comment]
There is a setting for the witchwood biomes to change there rarity.

**Comment by Webber88** [comment]
i changed tht... im currently compiling a list of mods im using once im done ill upload a link .

**Comment by Webber88** [comment]
heresa link to a page with the mods i have installed(had to upload it to dropbox sorry)
https://www.dropbox.com/sh/b0b61rsr0ypz9mw/AADqKM_b3YtUBt8PSd9lyyica?dl=0

**Comment by Webber88** [comment]
so would any of these mods cause the witchwood biome not to spawn?

**Comment by bookerthegeek** [comment]
Not really... just a little bit of hate from the rng. 

Do me a favor and install anti Id conflict and make site you have no biome id conflicts please.

**Comment by Webber88** [comment]
i installed an anti id conflict and i even jacked up the rarity(to make it mor common) to 9999999 and still nothing

**Comment by Webber88** [comment]
ok so i turned all my other mods off except for am2 and started a new world and still no witch wood forest biome

**Comment by bookerthegeek** [comment]
Yes, but the biomes are huge so you will be looking at more area to cover.

**Comment by jjtParadox** [comment]
@Webber88 any update for this?
